tests/pauli/dla/test_structure_constants.py

```diff
@@ -84,13 +84,14 @@ def test_structure_constants_elements_with_orthogonal_basis(
 
     @pytest.mark.parametrize("ortho_dla", [Ising3, XXZ3])
     @pytest.mark.parametrize("matrix", [False, True])
-    def test_structure_constants_elements_with_non_orthogonal(self, ortho_dla, matrix):
+    def test_structure_constants_elements_with_non_orthogonal(self, ortho_dla, matrix, seed):
         r"""Test relation :math:`[i G_α, i G_β] = \sum_{γ=0}^{d-1} f^γ_{α,β} iG_γ_` with
         non-orthogonal bases.
         """
         d = len(ortho_dla)
 
-        coeffs = np.random.random((d, d)) + 0.5
+        rng = np.random.default_rng(seed)
+        coeffs = rng.random((d, d)) + 0.5
```

Comment:
```suggestion
        coeffs = rng.uniform(0.5, 1.5, size=(d, d))
```
The test seems to have been extracted from a concrete physical context so my suggestion here might be improper. Just thought that if we don't have a specific implication for the shift 0.5 here  then probably it would be better to explicitly say `uniform(low, high, size=size` here. Personally whenever I use `np.random` module if it's `random.random` I have to convince myself that it is a uniform dist other than anything else lol.